tests/unit/helpers.js

```diff
@@ -701,3 +701,44 @@ export const createFakeTracking = (overrides = {}) => {
     ...overrides,
   };
 };
+
+// Save a reference to the real setTimeout in case a test uses a mock
+// sinon clock.
+const globalSetTimeout = setTimeout;
+
+/*
+ * Wait until the saga dispatches an action causing isMatch(action)
+ * to return true. Throw an error if it doesn't.
+ */
+export async function matchingSagaAction(
```

Comment:
I don't see any test case for this helper function.

Reply:
Whoops, good call. We should cover throwing on timeout.